HaloDAO/halodao-interface
# HDEV-245 Block finality progress countdown UI for the bridge

## Description of Changes
Implemented the UI for displaying the progress flow for block finality when using the bridge.
Figma here -> https://www.figma.com/file/lqpLD77mM2olJNrFzuD4Ul/HaloDAO-App?node-id=6302%3A2640

Note: 
This PR depends on branch on PR 103 https://github.com/HaloDAO/halodao-interface/pull/103
Please check PR 103 first

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-245)

### How to test
Note: Please test using xRNBW 
1. Go to bridge
2. Select any token that you'd like to bridge
3. Click "Approve" and/or "Next" 
4. Click confirm when the confirm modal pops out.
5. You should see the new progress UI flow indicated in Figma
6. For block Finality 
   6.1 when bridging from Polygon to Ethereum -> 192 blocks
  6.2 from Ethereum to Polygon -> 20 blocks
7.After finality, check if the correct balance for the token  is reflected


### Developer Checklist:


### Reviewers Checklist:
